sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-examples.adoc

```diff
@@ -1,161 +1,538 @@
-
-== <Example 2>
-
-// Same template for Example 1
-
-== <Example 3>
-.
-.
-.
+These examples show how to create flows to invoke some of SuccessFactor's standard operations, including how to:
+
+* <<create-query-flow,Create a query to query across entities>>
+* <<create-entry,Create an entry on SuccessFactors>>
+* <<get-entity-by-id,Get an entity by ID>>
+* <<update-entry,Create and update an entry on SuccessFactors>>
+* <<delete-entry, Delete an entry from SuccessFactors>>
+
+== Prerequisites
+
+To build and run these flows, you need:
+
+* OpenJDK 8 or 11
+* Anypoint Studio 7.1 or later
+* Mule 4.2.2 or later
+* SuccessFactors Connector 3.x or later
+* SuccessFactors credentials to send OData API requests
+* DataWeave 2.0
+
+=== Prerequisite Steps
+
+. <<create-mule-project,Create a Mule Project>>
+. <<specify-credentials,Specify Your Credentials>>
+. <<configure-global-element,Configure a Global Element>>
+
+[[create-mule-project]]
+=== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector: 
+
+. In Studio, select *File > New > Mule Project*.
+. In *Project Name*, enter a name for your Mule project. +
+This example uses the name `successfactors-operations`. 
+. Click *Finish*.
+
+=== Specify Your Credentials
+
+. In *Package Explorer*, under `/src/main/resources/`, create a file named `mule-app.properties`. 
+. Specify your credentials for connecting to SuccessFactors, for example:
++
+----
+config.companyId=<your-company-id>
+config.userName=<successfactors-username>
+config.password=<successfactors-password>
+----
++ 
+. Save the file. 
+
+[[configure-global-element]]
+=== Configure a Global Element for the Connector
+
+Configure a global element to provide the authentication credentials that the connector requires to access the target SAP SuccessFactors system.
+
+. At the base of the Studio canvas, click *Global Elements*. 
+. Select *SAP SuccessFactors Config* and click *Edit*. 
+. In the SAP SuccessFactors Configuration window, enter the connection credentials as follows:
+* *Company ID*: `${config.companyId}`
+* *User Name*: `${config.userName}`
+* *Password*:`${config.password}`
+. From the *Data Center's Endpoint URL* drop-down, select the SuccessFactor's Web API URL. 
++
+The following image shows an example of configuring Basic authentication:
++
+.Basic authentication fields
+image::sap-successfactors-basic-config.png
++
+. Click *OK*. 
+. Click *Test Connection* to verify the connection to SAP SuccessFactors.
++
+A success message appears.
+
+[[create-query-flow]]
+== Create the Query Operation Flow 
+
+Create a flow that uses the *Query* operation to perform queries over entities of a specified type:
+
+. In the *Mule Palette* view, select *HTTP > Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify `/list` for the *Path* field.
```

Comment:
Now I'm being really nitpicky: The other options state the location of the action before the action.

Reply:
This is the same location as the previous step. There is no need to specify it again. They are on the same screen.